frontend/src/components/pages/HomePage/LogRecordsTable.tsx

```diff
@@ -85,6 +85,31 @@ const LogRecordsTable = ({
     }));
   };
 
+  const formatNote = (note: string) => {
+    const noteLimit = 150;
```

Comment:
nit: Move this up higher in the file (reduces memory of function invocation) + renaming it to adhere to other constant names

```suggestion
    const NOTE_LIMIT = 150;
```